locales/en.json

```diff
@@ -837,6 +837,9 @@
     "holdings": "Holdings",
     "gains": "Gains",
     "showQr": "Show QR",
+    "searchNoResultsTitle": "No Assets On Selected Chain",
+    "searchNoResultsText": "Explore The Artist's Protfolio On Other Networks ({0})",
```

Comment:
```suggestion
    "searchNoResultsText": "Explore The Artist's Portfolio On Other Networks ({0})",
```

Reply:
remove "The"?
btw why everything is capitalize 😭 ?

Reply:
we can make it normal - its from the figma text style i suppose

Reply:
> remove "The"?
OK

> btw why everything is capitalize 😭 ?
It's just another solution to apply `capitalize` css style.

Reply:
> we can make it normal - its from the figma text style i suppose

Alright